### PR TITLE
export SUSEConnect files if present on the host

### PR DESCRIPTION
**Pull request by fcrozat**

### CODE REVIEW

File: toolbox [hunk 1]
```diff
@@ -165,6 +165,10 @@ list() {
 }
 
 container_create() {
+    VOLUMES=""
```
Comment (dfaggioli) [inline toolbox:168]: While we do this, I think we could put all the `--volume`-s in a variable. But I can do it myself in another patch

**Review by dfaggioli — COMMENTED** [review]
This looks fine to me, in general. However, this is what I see in openSUSE MicroOS:

WARN[0000] Path "/etc/SUSEConnect" from "/etc/containers/mounts.conf" doesn't exist, skipping 
WARN[0000] Path "/etc/zypp/credentials.d/SCCcredentials" from "/etc/containers/mounts.conf" doesn't exist, skipping

So, doesn't this mean podman is already trying to do automatically what this patch does manually?

**Comment by jlausuch** [comment]
We already have this configuration in 

```
localhost:~ # cat /etc/containers/mounts.conf 
# This configuration file specifies the default mounts for each container of the
# tools adhering to this file (e.g., CRI-O, Podman, Buildah).  The format of the
# config is /SRC:/DST, one mount per line.
/etc/SUSEConnect:/etc/SUSEConnect
/etc/zypp/credentials.d/SCCcredentials:/etc/zypp/credentials.d/SCCcredentials
```
 So, I think this is not needed. Actually, this configuration is also in TW and it fails cause the directories don't exist (obviously).
 But for SLE Micro it just doesn't work, it does not mount them. Not sure why, maybe older podman vesion than in TW?

**Comment by fcrozat** [comment]
I doesn't work for SLE Micro nor SLE15 SP2

**Comment by thkukuk** [comment]
No activity and I don't think we need that,, as this files should be auto-mounted.

**Pull request closed without merging**